examples/with-auth/pages/auth/signin.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import withSession from '../../components/with-session'
+import Session from '../../components/session'
+
+class SignIn extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      email: ''
+    }
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.handleEmailChange = this.handleEmailChange.bind(this)
+  }
+
+  handleEmailChange (event) {
+    this.setState({email: event.target.value.trim()})
+  }
+
+  async handleSubmit (event) {
+    event.preventDefault()
+
+    const session = new Session()
+    session.signin(this.state.email)
+    .then(() => {
+      this.props.url.push('/auth/check-email')
+    })
+    .catch(err => {
+      // @FIXME Handle error
+      console.log(err)
+    })
+  }
+
+  render () {
+    let signinForm = <div />
+    if (!this.props.session.user) {
+      signinForm = (
+        <div>
+          <form id='signin' method='post' action='/auth/email/signin' onSubmit={this.handleSubmit}>
+            <input name='_csrf' type='hidden' value={this.props.session.csrfToken} />
```

Comment:
is this hidden input necessary with this already in place? https://github.com/zeit/next.js/pull/1141/files#diff-cba823cf5d072f066058359e8242073aR145